src/api/java/CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ set(JAVA_FILES
 
 # specify generated jni headers
 set(JNI_HEADERS
-  ${JNI_DIR}/cvc5_Solver.h
+  ${JNI_DIR}/io_github_cvc5_api_Solver.h
```

Comment:
I forgot why this was added. Is this variable really needed?

Reply:
This variable is used as the `OUTPUT` of the custom command behind the `generate-jni-headers` target and the `DEPENDS` of said target.